app/models/user.rb

```diff
@@ -355,15 +355,19 @@ def vet360_contact_info
     @vet360_contact_info ||= VAProfileRedis::ContactInformation.for_user(self)
   end
 
+  def va_profile_email
+    vet360_contact_info&.email&.email_address
+  end
+
   def all_emails
-    vet360_email =
+    the_va_profile_email =
       begin
-        vet360_contact_info&.email&.email_address
+        va_profile_email
       rescue
         nil
       end
 
-    [vet360_email, email]
+    [the_va_profile_email, email]
```

Comment:
```suggestion
    [va_profile_email, email]
```

I think you can just get rid of 363-369.